go/vt/discovery/healthcheck.go

```diff
@@ -427,6 +433,28 @@ func (hc *HealthCheckImpl) servingConnStats() map[string]int64 {
 	return res
 }
 
+// stateChecksum returns a crc32 checksum of the healthcheck state
+func (hc *HealthCheckImpl) stateChecksum() int64 {
+	status := hc.cacheStatusMap()
+	keys := make([]string, 0, len(status))
+	for key := range status {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+
+	var buf bytes.Buffer
+	for _, key := range keys {
+		// key contains Cell/Keyspace/Shard/TabletType
+		val := status[key]
+		buf.WriteString(key)
+		for _, ts := range val.TabletsStats {
```

Comment:
Is `val.TabletStats` deterministically sorted?

Reply:
Nope, good catch. Will need to sort.